### PR TITLE
Update docs when generating new Datasource

### PR DESCRIPTION
**Pull request by blackboxsw**
Minor updates to docs to tell DS creators they need to add dsname entries to datasource_dsname_map.rst

## Proposed Commit Message
See individual commits

## Merge type

- [ ] Squash merge using "Proposed Commit Message"
- [x] Rebase and merge unique commits. Requires commit messages per-commit each referencing the pull request number (#<PR_NUM>)

### CODE REVIEW

File: doc/rtd/development/datasource_creation.rst [hunk 1]
```diff
@@ -158,10 +158,11 @@ packaging configuration.
...
 
-You should add a new file in
-:file:`doc/rtd/reference/datasources/<cloudplatform>.rst`
-and reference it in
-:file:`doc/rtd/reference/datasources.rst`
+You should update the following docs:
+1. Add a new file in :file:`doc/rtd/reference/datasources/<cloudplatform>.rst`
+2. Reference `<cloudplatform>.rst` in :file:`doc/rtd/reference/datasources.rst`
+3. Add an alphebetized dsname entry in representing your datasource
+:file:`doc/rtd/reference/datasource_dsname_map.rst`
```
Comment (a-dubs) [inline doc/rtd/development/datasource_creation.rst:165]: should there be a more direct call to action here to use the command found in `doc/rtd/reference/datasource_dsname_map.rst` to generate the datasource list? Or is it just easier to tell the user to simply insert their new DS name in the list alphabetically (like you currently have)?

**Review by a-dubs — APPROVED** [review]
Looks good! Have one question but it is not blocking and more of a nit/potential suggestion so not gonna block on it

File: doc/rtd/development/datasource_creation.rst (hunk 1 above)
Comment (TheRealFalcon) [reply doc/rtd/development/datasource_creation.rst:165]: It shouldn't need to be regenerated each time, so I think the instructions here make the most sense.

**Review by TheRealFalcon — APPROVED** [review]
LGTM!

**Pull request merged**